src/System.Net.WebSockets.Client/src/System/Net/WebSockets/WinRTWebSocket.cs

```diff
@@ -515,5 +531,29 @@ private void UpdateStateCloseReceived()
             }
         }
         #endregion
+
+        #region Helpers
+        private static bool InitMessageWebSocketReceiveModeSupported()
+        {
+            return ApiInformation.IsPropertyPresent(
+                "Windows.Networking.Sockets.MessageWebSocketControl",
+                "ReceiveMode");
+        }
+
+        private bool IsPartialMessageEvent(MessageWebSocketMessageReceivedEventArgs eventArgs)
+        {
+            // The MessageWebSocketMessageReceivedEventArgs.IsMessageComplete property was introduced in the same
+            // Windows SDK flight as MessageWebSocketControl.ReceiveMode.
```

Comment:
I don't think we need this comment.  It is transitory because it talks about the "flight". Once the final next release of Windows is finished, the comment won't make as much sense.  I would delete this whole comment since the `if (MessageWebSocketReceiveModeSupported)` is sufficient to document that this is runtime detection of features.